rod_cutting_problem/rod_cutting.py

```diff
@@ -0,0 +1,35 @@
+"""
+    This module calculates optimized solution for rod cutting
+    by function rod_cutting() with arguments as defined
+    in main()
+"""
+
+
+def rod_cutting(price):
+    """
+    Computes maximum money that can be earned by cutting
+    a rod of length len(price) (Bottom-Up Approach).
+
+    Time Complexity : O((len(price))^2)
+    Space Complexity : O(len(price))
+    :param price: List in which price[i] denotes price of rod of length i.
+    :return: returns optimal solution for rod of length len(price).
+    """
+    opt_price = [0] * (len(price) + 1)
```

Comment:
You can avoid calculating length of `price` again and again by declaring it as some variable.